packages/frontend/src/components/create-campaign/form/kpi-step/simulation-chart/tooltip/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { formatPercentage, formatUsdAmount } from "@/src/utils/format";
+import styles from "./styles.module.css";
+import { useTranslations } from "next-intl";
+import { Typography } from "@metrom-xyz/ui";
+import { ReferenceLine } from "recharts";
+import type { ChartData } from "..";
+
+interface TooltipProps {
+    totalRewardsUsd: number;
+    lowerUsdTarget: number;
+    upperUsdTarget: number;
+    minimumPayoutPercentage: number;
+}
+
+interface TooltipCursorProps {}
```

Comment:
Also here, no way to type the props?

Reply:
Fixed in 0edb2c51d297f15fb54f2bfbba97283afc4c78e6.